src/itree.cpp

```diff
@@ -122,6 +122,19 @@ int main(int argc, char** argv){
         auto path = correct_path(vm, env);
         irods::experimental::client_connection conn;
 
+        const auto s = fs::client::status(conn, path);
+
+        if (!fs::client::exists(s))
+        {
+            std::cerr << fmt::format("Error: logical path [{}] does not exist.",path.c_str()) << std::endl;
+            return 1;
+        }
+        else if (fs::client::is_data_object(s))
+        {
+            std::cerr << fmt::format("Error: logical path [{}] refers to a data object, but itree works on collections only.",path.c_str()) << std::endl;
```

Comment:
Remove `std::endl` and add a `\n` to the format string.

I think `std::cerr` flushes automatically. But if it doesn't, we still avoid the flush due to the use of `std::endl`.